src/components/Form.js

```diff
@@ -61,15 +67,36 @@ function Form({
     }
   };
 
+  const handleSelectedCityFromList = (e, query) => {
+    e.preventDefault();
+
+    const city = `${query.fields.city}, ${query.fields.country}`;
+
+    const url = forecast ? `https://api.weatherbit.io/v2.0/forecast/daily?city=${city}&key=${fkey}` : `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${wkey}&units=metric`;
+
+    if (hourly) {
+      onFetchHourlyByName(city, fkey, wkey);
+    } else {
+      onFetchWeatherByName(forecast, url);
+    }
+
+    setSearchedQuery('');
+    setSelectedCity(null);
+  };
+
   return (
-        <FormContainer onSubmit={handleSearchQuery}>
+        <FormContainer id='form' onSubmit={selectedCity ? (e) => handleSelectedCityFromList(e, selectedCity) : handleSearchQuery}>
```

Comment:
Please use double quotation marks for React/html element's attributes.

Reply:
Fixed